### ISSUE TITLE
Question, how do you generate and test the Python examples in the README? 

### ISSUE THREAD

**Issue by shner-elmo** [pola-rs/polars#17309]
### Description

Hey, this is not an issue, just a question. Was wondering how you generate the examples in the README, and if (and how) you test them in the CI? (I imagine you need to manually parse the `>>>` and `... `)
I would appreciate if you could point me to the code that handles this, thanks.

(like [this](https://github.com/pola-rs/polars?tab=readme-ov-file#python) for example)

### Link

https://github.com/pola-rs/polars?tab=readme-ov-file#python

**Comment by mcrumiller** [comment]
Good point, that should be fixed. I'll explain why it looks like that though. If you enter a multi-line command into the python prompt, that's what you see:

![image](https://github.com/pola-rs/polars/assets/1896992/e0fafada-acfe-4271-b689-25f4691f41e9)

In the codebase, the function docstrings show these extra characters so as to mimic would it looks like running in terminal, but the html renders such that those characters aren't actually copied. See the example in [`keep_name](https://docs.pola.rs/api/python/stable/reference/expressions/api/polars.Expr.keep_name.html): if you copy the code, only the relevant code portions are selectable:

![image](https://github.com/pola-rs/polars/assets/1896992/41fa55a7-07db-4913-abe1-c28cb5949376)

It was probably intended to have the UI work this same way too, but it currently doesn't in the README.

**Comment by shner-elmo** [comment]
So if I understand correctly, the examples in the README are written by hand and are not tested in the CI as of yet.

But I wonder how you can change the CSS of the text that matches a pattern, like `^(?:\.\.\.|>>>) (.*)$` to modify those characters to be non-selectable? not sure if its possible
